packages/react-magma-dom/src/components/Tabs/TabsScrollSpyContainer.test.js

```diff
@@ -0,0 +1,190 @@
+    });
+
+    it('Should set the window.location.href on click', () => {
+      const { container } = render(
+        <TabsScrollSpyContainer testId={testId}>
+          <TabScrollSpyPanel tabLabel={'first'}>
+            Content Area One
+          </TabScrollSpyPanel>
+        </TabsScrollSpyContainer>
+      );
+
+      const tab3 = container.querySelector('button[data-scrollspy-id="first"]');
+
+      fireEvent.click(tab3);
+
+      expect(window.location.href).toInclude('#first');
+    });
+
+    it('Should prevent disabled from being clickable', () => {
+      const { container } = render(
+        <TabsScrollSpyContainer testId={testId}>
+          <TabScrollSpyPanel tabLabel={'first'} disabled>
+            Content Area One
+          </TabScrollSpyPanel>
+        </TabsScrollSpyContainer>
+      );
+      const tab1 = container.querySelector('button[data-scrollspy-id="first"]');
+
+      expect(tab1).toHaveAttribute('disabled');
+    });
+  });
+
+  it('Tab Panels are compliant with accessibility', () => {
+    const { container } = render(
+      <TabsScrollSpyContainer aria-label="TabsScrollSpyContainer">
+        <TabScrollSpyPanel tabLabel={'first'}>{TEXT}</TabScrollSpyPanel>
+      </TabsScrollSpyContainer>
+    );
+    return axe(container.innerHTML).then(result => {
+      return expect(result).toHaveNoViolations();
+    });
+  });
+});
+
+describe('TabScrollSpyPanel', () => {
+  it('Should find element by testId', () => {
+    const { getByTestId } = render(
+      <TabScrollSpyPanel tabLabel={tabLabel} testId={testId} />
+    );
+
+    expect(getByTestId(testId)).toBeInTheDocument();
+  });
+
+  it(`Should set the navigation title with 'tabLabel'`, () => {
+    const { getByTestId } = render(
+      <TabsScrollSpyContainer testId={`${testId}-Tab`}>
+        <TabScrollSpyPanel tabLabel={`${tabLabel}-1`} />
+        <TabScrollSpyPanel tabLabel={`${tabLabel}-2`} />
+      </TabsScrollSpyContainer>
+    );
+
+    expect(getByTestId(`${testId}-Tab`)).toHaveTextContent(`${tabLabel}-1`);
+    expect(getByTestId(`${testId}-Tab`)).toHaveTextContent(`${tabLabel}-2`);
+  });
+
+  it(`Should set an icon in each navigation item with 'icon'`, () => {
+    const { getByTestId } = render(
+      <TabsScrollSpyContainer testId={`${testId}-Tab`}>
+        <TabScrollSpyPanel tabLabel={tabLabel} icon={<AndroidIcon />} />
+      </TabsScrollSpyContainer>
+    );
+
+    expect(
+      getByTestId(`${testId}-Tab`).querySelector('svg')
+    ).toBeInTheDocument();
+  });
+
+  it(`Should set a disabled state in each navigation item with 'disabled'`, () => {
+    const { container } = render(
+      <TabsScrollSpyContainer>
+        <TabScrollSpyPanel tabLabel="disabled-nav" disabled>
+          {TEXT}
+        </TabScrollSpyPanel>
+      </TabsScrollSpyContainer>
+    );
+
+    const tab1 = container.querySelector('button[data-scrollspy-id="first"]');
+
+    expect(tab1.parentElement).toHaveStyleRule('cursor', 'not-allowed');
```

Comment:
I'm confused how this test passes but the `TEXT` is not first, so how is it getting that id?

Reply:
`disabled` removed entirely